exchanges/okx/okx_test.go

```diff
@@ -4025,6 +4025,14 @@ func TestPushData(t *testing.T) {
 	t.Parallel()
```

Comment:
```suggestion
	t.Parallel()
	ok := new(Okx) //nolint:govet // Intentional shadow to avoid future copy/paste mistakes
	require.NoError(t, testexch.Setup(ok), "Setup must not error")

	for x := range pushDataMap {
		if x == "Balance And Position" {
			ok.API.AuthenticatedSupport = true
			ok.API.AuthenticatedWebsocketSupport = true
			ok.SetCredentials("test", "test", "test", "", "", "")
		} else {
			ok.API.AuthenticatedSupport = false
			ok.API.AuthenticatedWebsocketSupport = false
		}
		err := ok.WsHandleData([]byte(pushDataMap[x]))
		require.NoErrorf(t, err, "Okx %s error %v", x, err)
	}
```

This will need a local instance since ATM there's a lot of test failures as it's overwriting the shared OKX instance

Reply:
Fixed.